source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+            deletes:
+              - q: {}
+                limit: 0
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: default_write_concern_coll
+            documents:
+              - _id: 1
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: default_write_concern_coll
+            updates:
+              - q:
+                  _id: 1
+                u:
+                  $set:
+                    x: 1
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: default_write_concern_coll
+            documents:
+              - _id: 2
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: default_write_concern_coll
+            updates:
+              - q:
+                  _id: 1
+                u:
+                  x: 2
+            writeConcern: null
+      - command_started_event:
+          command:
+            insert: default_write_concern_coll
+            documents:
+              - _id: 3
+            writeConcern: null
+      - command_started_event:
+          command:
+            update: default_write_concern_coll
+            updates:
+              - q:
+                  _id: 1
+                u:
+                  $set:
+                    x: 3
+                multi: true
+            writeConcern: null
+      - command_started_event:
+          command:
+            delete: default_write_concern_coll
+            deletes:
+              - q:
+                  _id: 3
+                limit: 1
+            writeConcern: null
+  - description: 'findOneAnd{update,delete,replace}'
+    operations:
+      - name: findOneAndUpdate
+        arguments:
+          filter:
+            _id: 1
+          update:
+            $set:
+              x: 1
+      - name: findOneAndReplace
+        arguments:
+          filter:
+            _id: 2
+          replacement:
+            x: 2
+      - name: findOneAndDelete
+        arguments:
+          filter:
+            _id: 2
+    outcome:
+      collection:
+        name: default_write_concern_coll
+        data:
+          - _id: 1
+            x: 1
+    expectations:
+      - command_started_event:
+          command:
+            findAndModify: default_write_concern_coll
+            query:
+              _id: 1
+            update:
+              $set:
+                x: 1
+            writeConcern: null
+      - command_started_event:
+          command:
+            findAndModify: default_write_concern_coll
+            query:
+              _id: 2
+            update:
+              x: 2
+            writeConcern: null
+      - command_started_event:
+          command:
+            findAndModify: default_write_concern_coll
+            query:
+              _id: 2
+            remove: true
+            writeConcern: null
+  - description: 'insert{one,many}'
```

Comment:
"InsertOne and InsertMany omit default write concern" may read better for drivers reporting descriptions.

Reply:
Done.